nltk/internals.py

```diff
@@ -411,7 +412,7 @@ def get(self):
 ##########################################################################
 
 def find_file_iter(filename, env_vars=(), searchpath=(),
-        file_names=None, url=None, verbose=True):
+        file_names=None, url=None, verbose=True, finding_dir=False):
```

Comment:
What do you use to indent the code? It looks that use tabs, please use 4 spaces.